packages/render/package.json

```diff
@@ -54,4 +56,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
\ No newline at end of file
+}
```

Comment:
Can you run `yarn format`?

Reply:
Sure, ran the command but it seemed to format a bunch of files outside of the scope of this change (mostly in `/demo`) - removed the extra line here but not sure prettier supports formatting json though